### ISSUE TITLE
CSS not loading after changing the port in Nginx

### ISSUE THREAD

**Issue by hamedprog** [openspeedtest/Speed-Test#136]
When I clone the project and run it using Nginx, everything works fine. However, after changing the port to a value other than 3000, the site fails to display correctly, and an SVG file is downloaded containing all the site’s graphical elements.

Interestingly, when I tested the following downloadable version from the website: “Download Nginx-Server For Windows With OST Source” The site displays correctly even after changing the port. In the “ost” package, the “asset” folder is missing, and its subfolders are located in the project’s root directory. [Additionally, after copying the “html” folder from “ost” and transferring it to my own repository (which is a clone of the “openspeedtest” repo), changing the port still works as expected]

**Comment by openspeedtest** [comment]
@hamedprog once you've changed the port number, try accessing the application through an incognito window.

**Comment by hamedprog** [comment]
> @hamedprog once you've changed the port number, try accessing the application through an incognito window.

No, the problem was not solved with these changes. The issue occurs during the loading time of the SVG file. [I have submitted a pull request for this problem.](https://abadis.ir/translator/)
